.prettierrc

```diff
@@ -0,0 +1,2 @@
+# https://prettier.io/docs/en/options.html
+singleQuote: true
```

Comment:
I love using Prettier. I added this to keep the style of what's already been built ❤️ since Prettier defaults to double-quotes by default.

Reply:
FWIW, my editor hates this file as soon as I open it, but putting the prettier config in the package.json as

`"prettier": {
      "singlequote": "true"
}`

was accepted without error. Good call though.

Reply:
This file is in YAML format. If you set your editor to interpret all `.prettierrc` files as YAML, that should fix it.